### PR TITLE
Allow customization of sign request type of S3Uploader

### PR DESCRIPTION
**Pull request by quaertym**

### CODE REVIEW

File: packages/ember-uploader/lib/s3.js [hunk 1]
```diff
@@ -49,11 +50,17 @@ export default Uploader.extend({
...
+    if (signRequestType !== 'GET') {
+      data = JSON.stringify(data);
+    }
+
     var settings = {
       url: get(this, 'url'),
       headers: get(this, 'headers'),
-      type: 'GET',
-      contentType: 'json',
+      type: signRequestType,
```
Comment (digitaltoad) [inline packages/ember-uploader/lib/s3.js]: Should we make sure this is a valid type?

Comment (quaertym) [inline packages/ember-uploader/lib/s3.js]: I think jQuery will use the default value.

Comment (digitaltoad) [inline packages/ember-uploader/lib/s3.js]: Quick testing says it doesn't work that way, however, after thinking more, I'm not so sure its a great thing to cover all method types. If jQuery doesn't cover it, we probably shouldn't either.

Comment (digitaltoad) [inline packages/ember-uploader/lib/s3.js]: I would also note, I'm not covering this in other areas.

**Comment by digitaltoad** [comment]
Can you go ahead and regenerate with this and I'll keep it under the same pr?

**Comment by quaertym** [comment]
Can you clarify what do you mean by `regenerate with this`? Do you mean that we should add a comment?

**Comment by digitaltoad** [comment]
I meant rebuild but I can do it in the next commit.

**Comment by quaertym** [comment]
Oh, I see. I will leave that to you. Maybe you can release a new version.

**Comment by quaertym** [comment]
Can you merge and release today? I am eager to use it.

**Pull request merged**